### PR TITLE
Integrate 12/2 Nightly Build

### PR DESCRIPTION
**Pull request by NickGerleman**
Commits: https://github.com/facebook/react-native/compare/19d4cc2d4...c901c1fbc

Interesting changes for us are changes to accommodate the new version of Metro brought in by the CLI, ScrollResponder fixes, bundling changes when using Hermes. New metro also breaks windesktop/desktop integration tests. Need to figure out what to do there.

There were a whole bunch of ScrollView changes but I don't think any of them should affect us.

Wanting to take a closer look at ViewConfig refactoring soon since I think more may be exposed than previously.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/6663)

### CODE REVIEW

File: vnext/PropertySheets/Bundle.props [hunk 1]
```diff
@@ -36,7 +36,8 @@
     <UseDevBundle Condition="'$(UseDevBundle)' == '' and '$(Configuration)' != 'Debug'">false</UseDevBundle>
 
     <!-- Extra arguments to pass to the bundler command -->
-    <BundlerExtraArgs Condition="'$(BundlerExtraArgs)' == ''"></BundlerExtraArgs>
+    <BundlerExtraArgs Condition="'$(BundlerExtraArgs)' == '' AND '$(UseHermes)' != 'true'"></BundlerExtraArgs>
+    <BundlerExtraArgs Condition="'$(BundlerExtraArgs)' == '' AND '$(UseHermes)' == 'true'">--minify false</BundlerExtraArgs>
```
Comment (dannyvv) [inline vnext/PropertySheets/Bundle.props:40]: >minify [](start = 93, length = 6)

Is this intentional? Shouln't it be debug only?

File: vnext/template/metro.config.js [hunk 2]
```diff
@@ -5,11 +5,11 @@
  * @format
  */
 const path = require('path');
-const blacklist = require('metro-config/src/defaults/blacklist');
+const exclusionList = require('metro-config/src/defaults/exclusionList');
```
Comment (dannyvv) [inline vnext/template/metro.config.js:8]: >exclusionList [](start = 57, length = 13)

This change will likely need updating in our documentation as well.

**Review by dannyvv — APPROVED** [review]
:shipit:

File: vnext/PropertySheets/Bundle.props (hunk 1 above)
Comment (NickGerleman) [reply vnext/PropertySheets/Bundle.props:40]: It's intentional. The reason is that Hermes will convert the JS bundle to bytecode, so minification doesn't affect the final output. Just wastes some cycles.

Not sure how much the optimization does, but we now do this for Android/iOS.

**Pull request merged**